templates/index.html

```diff
@@ -4,18 +4,18 @@
   <title>{{ config.title }}</title>
   <meta property="og:title" content="{{ config.title }}" />
   <meta property="og:description" content="{{ config.description }}" />
-  <meta property="og:url" content="https://limeleaf.net" />
-  <meta property="og:image" content="https://limeleaf.net/images/logo/limeleaf-logo.jpg" />
+  <meta property="og:url" content="https://limeleaf.coop" />
+  <meta property="og:image" content="https://limeleaf.coop/images/logo/limeleaf-logo.jpg" />
```

Comment:
These can become `{{ config.base_url }}/images/logo/limeleaf-logo.jpg` so we never have to change them again.